src/main/kotlin/com/mutualmobile/praxisspringboot/controllers/orgs/impl/UserProjectApiImpl.kt

```diff
@@ -34,4 +37,38 @@ class UserProjectApiImpl : UserProjectApi {
 
         return userProjectService.assignProjectToUser(projectId = projectId, userId = userId)
     }
+
+    override fun logWorkTime(
+        projectId: String,
+        workDate: Date,
+        workHours: Int,
+        note: String?,
+        httpServletRequest: HttpServletRequest
+    ): ResponseEntity<ApiResponse<Unit>> {
+        return try {
+            val userId =
+                userDataService.getUser(httpServletRequest)?.id
```

Comment:
this will never be the case @shubhamsinghmutualmobile but a good security check, although it's not needed since you will never find the project with that prodect id and user id.